Nanocloud/nanocloud
# Add Guacamole RDP options settable

Fixes #450 
All RDP options of Guacamole documented [here](http://guacamole.incubator.apache.org/doc/gug/configuring-guacamole.html#rdp) are now settable in Nanocloud with config variables.
We use the prefix `rdp` and the name of the option, all in camel case for the config variables.

Step to validate:

- Set for example `rdpWidth` and `rdpHeight` to 1600 and 800
- Launch desktop app in Nanocloud

Expected result: The VDI size is equal to 1600x800px.
